source/process/mobile-release-process.md

```diff
@@ -9,58 +9,137 @@ Notes:
+    
+### C. (T-minus 10 working days) Judgment Day
+
+Day when PM decides which major features are included in the release, and which are postponed.
+
+1. **(Team) Triage Meeting**:
+    - Begin daily triage of tickets
+2. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Verify all items in the last posted release checklist are complete
+    - Update Changelog PR based on what's in/out of the release
+    - Create meta issue for release in GitHub
+    - Post a link to Native Mobile Apps channel for query of remaining bugs in this release
+3. PM:
+    - Review the JIRA tickets remaining in the current release fix version and push those that won't make it to the next fix version
+
+### D. (T-minus 8 working days) Code Complete and Release Candidate Cut 
+
+**Stabilization** period begins when all features for release have been committed. During this period, only **bugs** can be committed to the release branch. Non-bug pull requests are tagged for next version. Exceptions can be made by the Release Manager during triage.
+
+1. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Verify all items in the last posted release checklist are complete
+    - Update the GitHub meta issue to include a link to the changelog on the documentation branch
+2. PM:
     - Check the minimum server version required and submit pull request to update in fastlane files
-2. Dev:
-    - Finish review of last pull requests for the current release and merge
-3. QA:
+3. Dev:
+    - Prioritize reviewing, updating, and merging of pull requests for current release until there are no more tickets in the pull request queue marked for the current release
+4. QA:
     - Confirm all pull requests merged into the current release have been tested
-4. Build:
+    - Ensure the release testing spreadsheet covers any changes and new features, and confirm known issues are listed in the relevant tests
+    - Assign each area of the spreadsheet to a team member and give the core team access permissions
+5. Build:
     - Create release branch for mattermost-mobile and mattermost-redux
     - Cut release candidate build
+6. Docs:
+    - Submit any remaining documentation PRs for product updates in the release
 
+### E. (T-minus 7 working days) Release Candidate Testing
 
-### B. (T-minus 7 working days) Release Candidate Testing
-
-1. QA:
+1. Release Manager:
+    - Post this checklist in Release Checklist channel
+    - Verify all items in the last posted release checklist are complete
+    - Update the GitHub meta issue:
+        - Post comments to the meta issue with approved fixes for the next builds
+        - Update download links and testing server links to the latest build
+    - Post list of tickets to be fixed to the Release Discussion channel
+    - Update Changelog for any new bug fixes
+2. QA:
     - Post release testing instructions and spreadsheet to Native Mobile Apps channel
```

Comment:
Testing instructions are posted into release discussion channel (people will see their assignments in the spreadsheet if they are on mobile duty)